Description: Remove obsolete macro and fix spelling mistake
 The macro IX is obsolete and not recognized anymore.  The patch deletes the
 corresponding line and also fixes a spelling mistake.
Author: Arno Onken <asnelt@asnelt.org>
Index: rrep-1.3.6/doc/rrep.in.1
===================================================================
--- rrep-1.3.6.orig/doc/rrep.in.1
+++ rrep-1.3.6/doc/rrep.in.1
@@ -1,4 +1,3 @@
-.IX Title "RREP 1"
 .TH RREP 1 "rrep @VERSION@" "User Commands"
 .nh
 .SH "NAME"
@@ -109,7 +108,7 @@ for matching.
 This option can be used to specify a pattern beginning with \fB\-\fR.
 .TP
 .BR \-h ", " \-\^\-help
-Display a help message that descibes the command line options and exit
+Display a help message that describes the command line options and exit
 afterwards.
 .TP
 .BR \-i ", " \-\^\-ignore\-case
